spec.emu

```diff
@@ -186,10 +186,14 @@ contributors: Google, Ecma International
         1. Set _displayNames_.[[Fallback]] to _fallback_.
         1. Set _displayNames_.[[Locale]] to the value of _r_.[[Locale]].
         1. Let _dataLocale_ be _r_.[[dataLocale]].
-        1. Let _dataLocaleData_ be _localeData_.[[_dataLocale_]].
-        1. Let _dataLocaleTypes_ be _dataLocaleData_.[[_type_]].
-        1. Let _fields_ be dataLocaleTypes.[[_style_]].
-        1. Set _displayNames_.[[Fields]] to _fields_.
+        1. Let _dataLocaleData_ be _localeData_.[[<_dataLocale_>]].
+        1. Let _types_ be _dataLocaleData_.[[types]].
+        1. Assert: _types_ is a record (see <emu-xref href="#sec-Intl.DisplayNames-internal-slots"></emu-xref>).
```

Comment:
I think `Record` is supposed to always be titlecase.

Reply:
done